robotiq_force_torque_sensor/src/rq_sensor_state.cpp

```diff
@@ -154,14 +154,27 @@ static void rq_state_start_stream()
  */
 static void rq_state_run()
 {
+	static unsigned char retries = 0;
+
 	rq_com_listen_stream();
 
 	if(rq_com_get_valid_stream() == false)
 	{
+		retries++;
+		printf("invalid stream : retry #%d\n", retries);
+
+		if (retries >= 100)
```

Comment:
The retry limit should be declared as a constant, with a meaningful name either at the top of this function or the top of this source file.